esp/templates/email/default_email_html.txt

```diff
@@ -0,0 +1,5 @@
+<html>
+{% autoescape off %}
+{{ msgbdy }}
+{% endautoescape %}
```

Comment:
Huh, ok, cool.  If by "email us if you want to unsubscribe" you intend for a human to handle it, that's fine; we can't really do an automated thing since the from/reply-to could be an arbitrary address controlled by the chapter.